pkgs/tools/misc/upterm/default.nix

```diff
@@ -1,18 +1,18 @@
 { lib
-, buildGoModule
+, buildGo118Module
 , fetchFromGitHub
 , installShellFiles
 }:
 
-buildGoModule rec {
+buildGo118Module rec {
   pname = "upterm";
-  version = "0.7.3";
+  version = "0.8.2";
 
   src = fetchFromGitHub {
     owner = "owenthereal";
     repo = "upterm";
     rev = "v${version}";
-    hash = "sha256-eEzFqKYhsG5e1DVLWJq08NM9xyfn1yPNV0NIgOErj4E=";
+    sha256 = "1yxj5c1gq6ph986zh1bgkmdwc56bdlxs7c9wifzfy7py7sr0bi95";
```

Comment:
Let's not rollback from hash to `sha256` please.

Reply:
How do I get the other format? I failed figuring this out.

Reply:
```
❯ nix hash to-sri --type sha256 1yxj5c1gq6ph986zh1bgkmdwc56bdlxs7c9wifzfy7py7sr0bi95
sha256-JcUFsj7+Hu++izyxozttyxTGW51vBfgNSvAa/AIrsvs=
```

Reply:
nix versions 2.4 and after give you sri hash by default fyi